src/Diagram.js

```diff
@@ -279,6 +283,7 @@ const Diagram = props => {
         containerRef={containerRef}
         renderPanAndZoomControls={props.renderPanAndZoomControls}
         scroll={state.scroll}
+        contentSpan={{ x: extremeX, y: extremeY }}
```

Comment:
can memoize `contentSpan` ?

Reply:
I think there is no need, as we are dependent upon x and y values directly in [PanAndZoomContainer](https://github.com/ujjawalpabreja99/react-virtualized-flowchart/pull/2/files#diff-fa50ff659ac0d0bad829809f3061f7a785c9852fd029b4dacad4215138058323)

Reply:
No need to memoize, we are dependent upon values of `x` and `y` only in underlying component.